libs/datamodel/core/src/validator/lower.rs

```diff
@@ -119,28 +120,29 @@ impl LowerDmlToAst {
 
     pub fn lower_default_value(dv: dml::DefaultValue) -> ast::Expression {
         match dv {
-            dml::DefaultValue::Single(v) => Self::lower_scalar_value(&v),
+            dml::DefaultValue::Single(v) => Self::lower_prisma_value(&v),
             dml::DefaultValue::Expression(e) => {
-                let exprs = e.args.iter().map(Self::lower_scalar_value).collect();
+                let exprs = e.args.iter().map(Self::lower_prisma_value).collect();
                 ast::Expression::Function(e.name, exprs, ast::Span::empty())
             }
         }
     }
 
-    pub fn lower_scalar_value(sv: &dml::ScalarValue) -> ast::Expression {
-        match sv {
-            dml::ScalarValue::Boolean(true) => ast::Expression::BooleanValue(String::from("true"), ast::Span::empty()),
-            dml::ScalarValue::Boolean(false) => {
-                ast::Expression::BooleanValue(String::from("false"), ast::Span::empty())
-            }
-            dml::ScalarValue::String(value) => ast::Expression::StringValue(value.clone(), ast::Span::empty()),
-            dml::ScalarValue::ConstantLiteral(value) => {
-                ast::Expression::ConstantValue(value.clone(), ast::Span::empty())
-            }
-            dml::ScalarValue::DateTime(value) => ast::Expression::ConstantValue(value.to_rfc3339(), ast::Span::empty()),
-            dml::ScalarValue::Decimal(value) => ast::Expression::NumericValue(value.to_string(), ast::Span::empty()),
-            dml::ScalarValue::Float(value) => ast::Expression::NumericValue(value.to_string(), ast::Span::empty()),
-            dml::ScalarValue::Int(value) => ast::Expression::NumericValue(value.to_string(), ast::Span::empty()),
+    pub fn lower_prisma_value(pv: &PrismaValue) -> ast::Expression {
+        match pv {
+            PrismaValue::Boolean(true) => ast::Expression::BooleanValue(String::from("true"), ast::Span::empty()),
+            PrismaValue::Boolean(false) => ast::Expression::BooleanValue(String::from("false"), ast::Span::empty()),
+            PrismaValue::String(value) => ast::Expression::StringValue(value.clone(), ast::Span::empty()),
+            PrismaValue::Enum(value) => ast::Expression::ConstantValue(value.clone(), ast::Span::empty()),
+            PrismaValue::DateTime(value) => ast::Expression::ConstantValue(value.to_rfc3339(), ast::Span::empty()),
```

Comment:
I'm wondering if datetimes should be quoted or not. I don't think I've seen datetimes as constants in a schema before.

Reply:
I adjusted it so they are lowered to string